build-info-extractor/src/main/java/org/jfrog/build/extractor/issuesCollection/IssuesCollector.java

```diff
@@ -62,8 +66,9 @@ IssuesCollectionConfig parseConfig(String config) throws IOException {
 
     /**
      * Gets the previous vcs revision from the LATEST build published to Artifactory.
-     * */
-    private String getPreviousVcsRevision(ArtifactoryBuildInfoClient client, String buildName) throws IOException {
+     */
+    private String getPreviousVcsRevision(ArtifactoryBuildInfoClientBuilder clientBuilder, String buildName) throws IOException {
+        ArtifactoryBuildInfoClient client = clientBuilder.build();
```

Comment:
Please close the client:
```suggestion
        try (ArtifactoryBuildInfoClient client = clientBuilder.build()) {
```